### DISCUSSION TITLE
2023春夏季开源操作系统训练营 每日/周学习实践过程记录

### DISCUSSION THREAD

**Discussion by chyyuu** [LearningOS/rust-based-os-comp2022#170] [General]
请参加训练营的同学吧自己的每日/周学习实践过程的链接发在此discussion下，便于大家相互学习。

**Comment by yukun-impa** [comment]
每日学习的Repo:
https://github.com/yukun-impa/LearningOS2023Record

**Comment by ChenMiaoi** [comment]
### The First Day I have learned: RUST
[The FIrst Day Repo](https://github.com/ChenMiaoi/NoteMarkdown/blob/main/Rust/Rust.md)

> Just learned aouble `complex` types in chapter 3. If there are mistakes, I hope you can correct it via contacting me and we can talk about. Thanks.

**Comment by DavinZhang** [comment]
Everyday learning repo posted here: https://github.com/DavinZhang/os-training-camp
Hope all of us can make it.

**Comment by ChenMiaoi** [comment]
### The Second Day I have learned: RUST
[The Second Day Repo](https://github.com/ChenMiaoi/NoteMarkdown/blob/main/Rust/Rust.md)

> Done the chapter 3 and 4
> If there are mistakes, I hope you can correct it via contacting me and we can talk about. Thanks.

**Comment by si-yuan-zhou** [comment]
https://github.com/si-yuan-zhou/rust_selflearning

**Comment by Lethephyr** [comment]
https://github.com/Lethephyr/OS_Camp_2023

**Comment by creatoy** [comment]
[学习记录](https://blog.creatio.top/docs/rcore_study/)
欢迎来交流

**Comment by WuTao18** [comment]
https://github.com/WuTao18/os-camp-2023-record

**Comment by kcxain** [comment]
仓库：https://github.com/kcxain/Kcx_Learning/tree/master/docs#schedule
在线网站：https://deconx.cn/#schedule

**Reply by MIracleyin** [reply]
blog 写的太好了！！

**Comment by why1te** [comment]
https://github.com/BayMax99/OS_record.git

**Comment by MIracleyin** [comment]
仓库：https://github.com/MIracleyin/LearningOS2023Record

**Comment by xiaqy71** [comment]
https://github.com/xiaqy71/os-comp2023-record

**Comment by wswtc** [comment]
https://github.com/wswtc/2023os

**Comment by Younggkid** [comment]
https://github.com/Younggkid/rcore-record

**Comment by zhaozihanzzh** [comment]
Repository: https://github.com/zhaozihanzzh/Rust-based-OS-comp-record

**Comment by zyr743325945** [comment]
https://github.com/zyr743325945/2023OS_Daily

**Comment by hexWars** [comment]
欢迎交流！
[blog](https://blog.sehnsucht.top/zh/posts/%E6%8A%80%E6%9C%AF/learningos%E5%AD%A6%E4%B9%A0%E6%97%A5%E5%BF%97/)
[repo](https://github.com/LearningOS/rust-rustlings-hexWars)

**Comment by 0130w** [comment]
学习记录repo，欢迎各位交流 :)
https://github.com/0130w/learningOsRecords

**Comment by Touring-686** [comment]
学习记录
https://github.com/Touring-686/OS-camp-record

**Comment by owlxiao** [comment]
学习记录
https://github.com/owlxiao/os-comp2023-record

**Comment by Jpz-sdju** [comment]
https://github.com/Jpz-sdju/os-comp-2023-learning

**Comment by duskgrow** [comment]
[学习日记](https://github.com/Lonjiazeng/Note/tree/master/Obsidian/Diary)

**Comment by jxd134** [comment]
学习记录（https://github.com/jxd134/rcore-record）

**Comment by starlitxiling** [comment]
每日学习记录
https://github.com/1754165401/rcore_learning_record

**Comment by CALMorACT** [comment]
学习记录
https://github.com/CALMorACT/LearningOSComp

**Comment by yl4869** [comment]
https://github.com/yl4869/record/blob/main/os_train/record.md

**Comment by MikeMMS** [comment]
https://github.com/MikeMMS/my_os_learning_record

**Comment by Ziqi-Yang** [comment]
https://github.com/Ziqi-Yang/LearningRustOS2023Record

**Comment by MoonOld** [comment]
daily record on git page https://moonold.github.io/posts/lab/rcore/record/
or repo https://github.com/MoonOld/rcore-record

**Comment by vanJker** [comment]
学习记录 repo
https://github.com/vanJker/os-comp2023

**Comment by uninfectable** [comment]
学习记录repo
https://github.com/uninfectable/LearningRust-OS2023

**Comment by GrittyB** [comment]
学习记录 repo:
https://github.com/GrittyB/rcore_os_study

**Comment by Humoooor** [comment]
每周学习记录 repo：
https://github.com/Humoooor/LearningOS_Comp2023_Record

**Comment by YJMSTR** [comment]
https://github.com/YJMSTR/LearningOS2023Records

**Comment by zxgsn** [comment]
https://github.com/zxgsn/LearningOS2023/blob/main/rust.md

**Comment by bigdaronlee163** [comment]
两周rust学习repo 包含8次提交。 
git log 如下：
![image](https://user-images.githubusercontent.com/49524138/232174030-e15dead0-80d2-41e1-9696-15234e994e7c.png)
https://github.com/LearningOS/rust-rustlings-bigdaronlee163

**Comment by WaterStop** [comment]
Rust学习总结：https://blog.csdn.net/qq_43840665/article/details/129899236
rustlings解析：https://blog.csdn.net/qq_43840665/article/details/130065993

话说有博客还要用gitpage吗🙄老师太忙，还是建一个吧。
https://github.com/WaterStop/rust-based-os-comp2023

**Comment by Takumi-wake** [comment]
更新每周学习repo
https://github.com/Takumi-wake/LearnOS_something_record

**Comment by iuqtcl** [comment]
https://github.com/iuqtcl/LearningOS2023Records

**Comment by doordie1117** [comment]
learning record：
https://github.com/doordie1117/rust-based-os-comp2023-learning-record

**Comment by catboomer1105** [comment]
学习repo：
https://github.com/3HYXX/LearningOS2023Record

**Comment by Unik-lif** [comment]
学习仓库：
https://eidos.gitbook.io/os-projects/
rCore其余资料：
https://github.com/Unik-lif/course_notes/tree/main/rCore

**Comment by Saltedf** [comment]
在线预览: https://saltedf.github.io/os-comp2023-record.html
仓库地址: https://github.com/Saltedf/saltedf.github.io/blob/main/os-comp2023-record.org

**Comment by Ingsuifon** [comment]
https://github.com/Ingsuifon/OS-LearningRecord

**Comment by lighkLife** [comment]
rCore 学习笔记：https://github.com/lighkLife/blog/issues

**Comment by yuhjvhhvj** [comment]
https://github.com/yuhjvhhvj/rcore-learning-blog
